docs/ugcstor-csi.md

```diff
@@ -0,0 +1,802 @@
+---
+id: ugcstor-csi
+title: cStor User Guide
+sidebar_label: cStor
+---
+------
+<br>
+
+<img src="/docs/assets/svg/3-config-sequence.svg" alt="OpenEBS configuration flow" style="width:100%">
+
+<br>
+<br>
+
+This user guide section provides the operations needed to be performed by the User and the Admin for configuring cStor related tasks. 
+
+   :::note 
+   The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
+   :::
+
+<h2> User operations</h2>
+
+[Snapshot and Clone of a cStor Volume](#snapshot-and-clone-of-a-cstor-volume)
+
+
+<h2> Admin operations</h2>
+
+[Creating cStor storage pools](#creating-cstor-storage-pool)
+
+[Creating cStor storage classes](#creating-cstor-storage-classes)
+
+[Expanding a cStor volume](#expanding-a-cstor-volume)
+
+[Performance Tunings in cStor Pools](#performance-tunings-in-cstor-pools)
```

Comment:
We can add volume tunnables as well using volume policy,  more info https://github.com/openebs/cstor-operators/blob/master/docs/tutorial/volumes/policy.md

Reply:
We will be adding these changes in the next PR.